### DISCUSSION TITLE
Error while starting zigbee-herdsman

### DISCUSSION THREAD

**Discussion by Landlieber** [Koenkk/zigbee2mqtt#20294] [General]
Hello,

I am trying to get Zigbee2Mqtt running, the whole thing was installed on a Windows 10 computer and the Sonoff Zigbee 3.0 USB DONGLE PLUS is used as an adapter.

I have followed the instructions: https://www.zigbee2mqtt.io/guide/installation/05_windows.html#determine-which-com-port-is-assigned-to-your-device

When I try to start Zigbee2Mqtt I get the following error message, does anyone know what I am doing wrong?

> PS C:\zigbee2mqtt-master> npm start
> 
> > zigbee2mqtt@1.34.0 start
> > node index.js
> 
> Zigbee2MQTT:info  2023-12-13 21:13:08: Logging to console and directory: 'C:\zigbee2mqtt-master\data\log\2023-12-13.21-13-08' filename: log.txt
> Zigbee2MQTT:info  2023-12-13 21:13:08: Starting Zigbee2MQTT version 1.34.0 (commit #unknown)
> Zigbee2MQTT:info  2023-12-13 21:13:08: Starting zigbee-herdsman (0.25.0)
> Zigbee2MQTT:error 2023-12-13 21:13:28: Error while starting zigbee-herdsman
> Zigbee2MQTT:error 2023-12-13 21:13:28: Failed to start zigbee
> Zigbee2MQTT:error 2023-12-13 21:13:28: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
> Zigbee2MQTT:error 2023-12-13 21:13:28: Exiting...
> Zigbee2MQTT:error 2023-12-13 21:13:29: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
>     at ZStackAdapter.start (C:\zigbee2mqtt-master\node_modules\zigbee-herdsman\src\adapter\z-stack\adapter\zStackAdapter.ts:103:27)
>     at Controller.start (C:\zigbee2mqtt-master\node_modules\zigbee-herdsman\src\controller\controller.ts:132:29)
>     at Zigbee.start (C:\zigbee2mqtt-master\lib\zigbee.ts:60:27)
>     at Controller.start (C:\zigbee2mqtt-master\lib\controller.ts:101:27)
>     at start (C:\zigbee2mqtt-master\index.js:107:5)

**Comment by Streamnet-coder** [comment]
i am facing similar error : 

[15:57:59] INFO: Preparing to start...
[15:57:59] INFO: Socat not enabled
[15:58:01] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:info  2023-12-19 15:58:07: Logging to console and directory: '/config/zigbee2mqtt/log/2023-12-19.15-58-07' filename: log.txt
Zigbee2MQTT:info  2023-12-19 15:58:07: Starting Zigbee2MQTT version 1.34.0 (commit #unknown)
Zigbee2MQTT:info  2023-12-19 15:58:07: Starting zigbee-herdsman (0.25.0)
Zigbee2MQTT:error 2023-12-19 15:58:28: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2023-12-19 15:58:28: Failed to start zigbee
Zigbee2MQTT:error 2023-12-19 15:58:28: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2023-12-19 15:58:28: Exiting...
Zigbee2MQTT:error 2023-12-19 15:58:28: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
    at ZStackAdapter.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/zStackAdapter.ts:103:27)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:132:29)
    at Zigbee.start (/app/lib/zigbee.ts:60:27)
    at Controller.start (/app/lib/controller.ts:101:27)
    at start (/app/index.js:107:5)
